### PR TITLE
feat(vbd): update valuebased.design with current product and adapters

### PR DESCRIPTION
**Pull request by zacjones93**
The stripe set up for this product was finished but as I was setting this up locally I updated adapters and product form/code to be in-sync with latest patterns ([work done](https://github.com/badass-courses/course-builder/pull/206) on `pro-nextjs` recently).

This also fixes [VBD-5](https://linear.app/skillrecordings/issue/VBD-5/login-shows-as-joel-instead-of-nick) because of a hardcoded value set for email sender in the email provider

gif

### CODE REVIEW

File: apps/value-based-design/next.config.mjs [hunk 1]
```diff
@@ -28,6 +28,11 @@ const config = {
 				hostname: 'image.mux.com',
 				port: '',
 			},
+			{
+				protocol: 'https',
+				hostname: 'res.cloudinary.com',
+				port: '',
+			},
```
Comment (zacjones93) [inline apps/value-based-design/next.config.mjs:35]: needed for product images, not sure why this wasn't an issue previously when I added cloudinary images for a lesson

File: apps/value-based-design/src/coursebuilder/email-provider.ts [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 import EmailProvider from 'next-auth/providers/nodemailer'
 
+import { sendVerificationRequest } from '@coursebuilder/core/lib/send-verification-request'
+
 export const emailProvider = EmailProvider({
-	from: 'joel@coursebuilder.dev',
+	from: env.NEXT_PUBLIC_SUPPORT_EMAIL,
+	sendVerificationRequest: (params) => {
+		return sendVerificationRequest(params, courseBuilderAdapter)
+	},
```
Comment (zacjones93) [inline apps/value-based-design/src/coursebuilder/email-provider.ts:11]: hardcoded email sender fixed

File: apps/value-based-design/src/coursebuilder/slack-provider.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { env } from '@/env.mjs'
+
+import SlackProvider from '@coursebuilder/core/providers/slack'
+
+export const slackProvider = SlackProvider({
+	apiKey: env.SLACK_TOKEN,
+	defaultChannelId: env.SLACK_DEFAULT_CHANNEL_ID,
+})
```
Comment (zacjones93) [inline apps/value-based-design/src/coursebuilder/slack-provider.ts:8]: dont think we have a slack channel set up for this but it's ready when we need it

File: apps/value-based-design/src/coursebuilder/stripe-provider.ts [hunk 4]
```diff
@@ -4,8 +4,8 @@ import { env } from '@/env.mjs'
 import StripeProvider from '@coursebuilder/core/providers/stripe'
 
 export const stripeProvider = StripeProvider({
-	errorRedirectUrl: `${env.COURSEBUILDER_URL}/checkout-error`,
+	errorRedirectUrl: `${env.COURSEBUILDER_URL}`,
 	baseSuccessUrl: `${env.COURSEBUILDER_URL}`,
-	cancelUrl: `${env.COURSEBUILDER_URL}/checkout-cancel`,
+	cancelUrl: `${env.COURSEBUILDER_URL}`,
```
Comment (zacjones93) [inline apps/value-based-design/src/coursebuilder/stripe-provider.ts:9]: there were now `checkout-error` or `checkout-cancel` pages so updated this to match other products

**Review by joelhooks — APPROVED** [review]

**Pull request merged**